modules/compute/ssh-public-keys/main.bicep

```diff
@@ -1,3 +1,9 @@
+metadata name = 'Public SSH Keys'
+metadata description = '''This module deploys a Public SSH Key.
```

Comment:
Just double checking if we want multiline descriptions and if this would be the format

Reply:
It would be, and we already have it in a few modules actually.